src/components/CommitteeTools/components/SelectKeyModal/index.scss

```diff
@@ -115,6 +116,9 @@
         }
         p {
           color: $dark-turquoise;
+          @media only screen and (max-width: $mobile-breakpoint) {
+            height: 40px;
+          }
```

Comment:
@mwogates I think it's better to be consistent with that and use the mobile breakpoint at the end of the file so we'll not have to use the media query more than once in the same file. See in example here: `src/components/CommitteeTools/index.scss`

Reply:
ok